microsoft-edge/progressive-web-apps-chromium/how-to/handle-protocols.md

```diff
@@ -39,9 +39,13 @@ Each entry in the `protocol_handlers` array contains a `protocol` string and a `
 
 In the above example, the app is registered to handle the `mailto` protocol.  When the operating system launches the app in response to the `mailto` protocol, the app navigates to the `/newEmail` URL, replacing the `%s` placeholder with the full URL that's being handled.
 
+## Test protocols with the DevTools
+
+You can use the **Application** tool in Microsoft Edge DevTools to verify that Microsoft Edge successfully registered your app as a handler for the protocols you defined in your web app manifest.  You can also test protocols and launch your PWA with different URLs and endpoints.  See [Test protocol handlers for PWAs](../../devtools-guide-chromium/progressive-web-apps/protocol-handlers.md) for more info.
```

Comment:
```suggestion
You can use the **Application** tool in Microsoft Edge DevTools to verify that Microsoft Edge successfully registered your app as a handler for the protocols you defined in your web app manifest.  You can also test protocols and launch your PWA with different URLs and endpoints.  For more information, see [Test protocol handlers for PWAs](../../devtools-guide-chromium/progressive-web-apps/protocol-handlers.md).
```
this comment only norms the link at end.  link text might need update per other comment